### PR TITLE
corrections of small mistakes in power plant description

### PR DESCRIPTION
**Pull request by sandrinecharousset**
Includes following corrections:
- traditionnal => traditional (for some instances)
- reservoir => Reservoir (for some instances)
- some run of river variables were wrongly named reservoir
- added missing variable turbine efficiency

### CODE REVIEW

**Review by erikfilias — APPROVED** [review]
Thank you, @sandrinecharousset, for the corrections.

**Comment by erikfilias** [comment]
The conflicts could be solved by a rebase of the branch, I think

**Comment by erikfilias** [comment]
This branch is outdated, no? Because #80 was created. If it is the case, please closes this PR.

**Comment by erikfilias** [comment]
Could you solve the conflict, please?

**Comment by sandrinecharousset** [comment]
> Could you solve the conflict, please? It could be made by a rebase or checking the power-plant.yaml.
> Thanks in advance.

I hope I did what was expected.....

**Review by erikfilias — APPROVED** [review]

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 1]
```diff
@@ -156,9 +156,9 @@ Number of Units|Electricity|Coal:
 Number of Units|Electricity|Lignite:
    description: Number of units of kind "Lignite power plant "
    unit:
-
-Maximum Active Power|Electricity|Biomass|Traditionnal:
-   description: Maximum active power of one typical Traditionnal Biomass power plant
+   
+Maximum Active power|Electricity|Biomass|Traditional:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:160]: Suggest minor editorial change to use only capitalized words in variable names
```suggestion
   
Maximum Active Power|Electricity|Biomass|Traditional:
```

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 2]
```diff
@@ -188,13 +188,13 @@ Maximum Active power|Electricity|Lignite:
    description: Maximum active power of one typical Lignite power plant
    unit: MW
 
-Minimum Active power|Electricity|Biomass|Traditionnal:
-   description: Minimum active power of one typical Traditionnal Biomass power plant
+Minimum Active power|Electricity|Biomass|Traditional:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:191]:
```suggestion
Minimum Active Power|Electricity|Biomass|Traditional:
```

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 3]
```diff
@@ -188,13 +188,13 @@ Maximum Active power|Electricity|Lignite:
...
+   description: Minimum active power of one typical Traditional Biomass power plant
                 biomass from local and pre-existing
                 resources mostly waste :domestic and agricultural waste, forest residues.
    unit: MW
 
-Minimum Active power|Electricity|Biomass|New and imported :
+Minimum Active power|Electricity|Biomass|New and imported:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:197]:
```suggestion
Minimum Active Power|Electricity|Biomass|New and imported:
```

**Review by danielhuppmann — CHANGES_REQUESTED** [review]
Thanks @sandrinecharousset for these clarifications, apologies for the late review - a few minor changes suggested, then good to be merged.

**Review by danielhuppmann — APPROVED** [review]
Thank you @sandrinecharousset!

**Pull request merged**